### DISCUSSION TITLE
Dropdown Changes

### DISCUSSION THREAD

**Discussion by HMM-MK** [filamentphp/filament#8339] [General]
In filament 2, I have invoice repeater form that have total cost field , I want to change format total cost field based user selected currency.

So, Text field::make()->dataList.....

How to know data changes option,

If user select USD , 

$set total_cost field = currency convert

How to make, Please help?

**Comment by leandrocfe** [comment]
You can use `->reactive()->afterStateUpdated(xx)`
https://filamentphp.com/docs/2.x/forms/advanced#updates
